tasks/main.yml

```diff
@@ -14,32 +14,39 @@
     mode: 0750
     state: directory
 
-- name: create local log dir
+- name: check playbook out
+  ansible.builtin.git:
+    repo: "{{ potos_specs.git_url }}/{{ potos_specs.git_repo }}.git"
```

Comment:
Introduces strong coupling to `potos_specs`?

Reply:
Yes, because playbook (defined in `potos_specs`) needs to be cloned, before creating a virtual environment in the same directory.

Added when clause to allow role usage without `potos_specs` being defined.